### ISSUE TITLE
\(...\) and \[...\] don't work

### ISSUE THREAD

**Issue by Pomax** [KaTeX/KaTeX#3855]
KaTeX for some bizarre reason still breaks on LaTeX math mode syntax, only supporting TeX math mode syntax, which is like saying "we support Italian" but only allowing Latin verb conjugations.  Supporting `$` and `$$` is handy, but those are the _plain TeX_ math mode indicators, _not_ the LaTeX indicators. Can you please finally after all these years fix this bug? So people can actually use KaTeX with properly written LaTeX that doesn't mix in ancient TeX syntax?

**Comment by edemaine** [comment]
Are you referring to auto-render? The default value for `delimiters` should include `$$...$$`, `\(...\)`, and `\[...\]`; in fact, it's sort of reversed: `$...$` isn't supported out of the box. See https://katex.org/docs/autorender

Perhaps you're in another context though? Could you specify?

**Comment by edemaine** [comment]
Indeed, the box is supposed to be math content. KaTeX only supports math mode, not outer par mode, so you're not supposed to include `$`s or `\(` or anything. Maybe "Type an expression" should say "Type a math expression"?

**Comment by Pomax** [comment]
A small change, but yeah I think that would probably go a long way to reduce confusion.
